src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ChangeLevelCalculator.kt

```diff
@@ -56,7 +56,7 @@ class ChangeLevelCalculator(
   }
 
   private fun hasNoAssessment(crn: String, offenderAssessment: OffenderAssessment?): Boolean =
-    (offenderAssessment == null).also { log.info("Valid assessment found for $crn : $it") }
+    (offenderAssessment == null).also { log.info("No valid assessment found for $crn : $it") }
```

Comment:
Should delete this really but it might be best to do all of that work at the same time